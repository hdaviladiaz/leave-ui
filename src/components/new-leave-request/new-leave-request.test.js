import React from 'react';
import NewLeaveRequest from './new-leave-request';
import renderer from 'react-test-renderer';

xit('should render NewLeaveRequest as snapshot', () => {
  const tree = renderer.create(<NewLeaveRequest/>).toJSON();
  expect(tree).toMatchSnapshot();
});
