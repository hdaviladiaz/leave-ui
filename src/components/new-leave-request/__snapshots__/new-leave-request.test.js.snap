// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NewLeaveRequest as snapshot 1`] = `
<div>
  <div
    className="form-group"
  >
    <label
      className="col-md-4 control-label"
      for="Thoughtworker "
    >
      Thoughtworker
    </label>
    <div
      className="col-md-4"
    >
      <input
        className="form-control input-md"
        id="Thoughtworker "
        name="Thoughtworker"
        placeholder="Thoughtworker"
        type="text"
      />
    </div>
  </div>
</div>
`;
